### PR TITLE
Issue/16 help page

### PR DESCRIPTION
**Pull request by bogdanadrianmarc**
This PR adds a help page to the app, according to issue https://github.com/FoodStandardsAgency/data-dot-food/issues/16 . It is accessed via the navigation bar and is currently just a draft, awaiting future relevant content

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "data-dot-food-editor",
-  "version": "1.0.6",
+  "version": "1.0.7",
```
Comment (ijdickinson) [inline package.json:3]: A new help section is a new feature, so this should be 1.1.0

File: src/Help.vue [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<template>
+  <main>
+    <div class="container">
+      <h1>Help</h1>
+      <h3>1. About this page</h3>
```
Comment (ijdickinson) [inline src/Help.vue:5]: Don't jump from `h1` to `h3`. If the page needs a visually smaller header font, use CSS

**Review by ijdickinson — COMMENTED** [review]
Looks good, but please fix the the h1/h3 issue. Thanks!

**Review by joescottdave — APPROVED** [review]
Looks ready to merge (and we await content)

**Pull request merged**